docs/learn-staking.md

```diff
@@ -137,10 +144,16 @@ GRANDPA and BABE equivocation slashing penalty is calculated as below:
 Validators may run their nodes on multiple machines to make sure they can still perform validation work in case one of their nodes goes down. It should be noted that if they do not have good coordination to manage signing machines, then equivocation is possible.
 
 > Notice:
-If a validator is reported for any one of the offences they will be removed from the validator set and they will not be paid while they are kicked out.
+If a validator is reported for any one of the offences they will be removed from the validator set ([chilled](#chilling)) and they will not be paid while they are kicked out.
 
 If you want to know more details about slashing, please look at our [research page](https://research.web3.foundation/en/latest/polkadot/slashing/amounts.html).
 
+### Chilling
+
+Chilling is the act of kicking a validator out of the active validator set, also disqualifying them from the set of electable candidates in the next NPoS cycle. This carries an implied penalty of being unable to earn rewards for the duration of the next era, and requires them to earn back any nominations they may have had before being kicked out.
+
+Because every offence regardless of level triggers a chill, and every chill triggers an automatic re-election of a new active validator set, the offending validator is immediately removed from the current set and the next, and someone else takes their place. To re-join, they need to acknowledge the slash and re-announce their candidature.
```

Comment:
I thought this changed when we had the large-scale slashing, and we had all the nominators chilled?  This caused some problems with keeping validators in the active set.

Reply:
If I'm reading [this](https://github.com/paritytech/substrate/blob/d940c02440f80c5e0ded092c4dbc9f048312d8a6/frame/staking/src/slashing.rs#L327) correctly, every offence chills, but some also slash?